FirebaseStorage/Sources/StorageComponent.swift

```diff
@@ -23,9 +23,6 @@ import FirebaseCore
 @objc(FIRStorageProvider)
 protocol StorageProvider {
   @objc func storage(for bucket: String) -> Storage
-  // TODO: See if we can avoid the `type` parameter by either making it a `Storage` argument to
```

Comment:
Comment was copied from FunctionComponent.swift, but not ever relevant for Storage.